azure-maven-plugin-lib/src/main/resources/MessageTemplates-aca.yaml

```diff
@@ -0,0 +1,169 @@
+---
+id:               configure-app-env-name
+promote:          "Input the Azure Container App Environment name (***${default_val}***):"
+default_val:      'env-${project.getArtifactId().replaceAll("[^-a-z0-9]", "").toLowerCase()}'
+resource:         Environment
+property:         environmentName
+expression:       true
+required:         true
+
+---
+id:               configure-resource-group-name
+promote:          "Input the resource group name (***${default_val}***):"
+default_val:      'rg-${project.getArtifactId().replaceAll("[^-a-z0-9]", "").toLowerCase()}'
+expression:       true
+required:         true
+resource:         Environment
+property:         resourceGroup
+
+---
+id:               configure-region
+promote:
+  header: "Regions:"
+  many: "Select region for Azure Container Apps:"
+message:
+  auto_select:    "There is only one valid region (***${ regions[0] }***) for your project, will use it automatically."
+  select_none:    "You have not selected any region, 'config' will terminate."
+required:         true
+auto_select:      true
+default_index:    0
+
+---
+id:               select-subscriptions
+promote:
+       header:    "Available subscription:"
+       many:      "Select subscription"
+required:         true
+auto_select:      true
+message:
+  empty_options:  "Cannot find any subscriptions."
+  auto_select:    "There is only one subscription (***${ subscriptions[0].displayName() }***) in your account, will use it automatically."
+  select_none:    "You have not selected any subscriptions, 'config' will terminate."
+default_index:    0
+
+---
+id:               select-ACA-env
+promote:
+  header:       "Available Azure Container App Environments:"
+  many:         "Select Azure Container App Environment for deployment:"
+message:
+  empty_options:  "There are no Azure Container App Environment in current subscription, please create it in Azure Portal."
+  auto_select:    "There is only one Azure Container App Environment (***${ environments[0].name() }***) in current subscription, will use it automatically."
+  select_none:    "You have not selected any Azure Container App Environment, 'config' will terminate."
+required:         true
+auto_select:      true
+default_index:    0
+
+---
+id:               select-app
+promote:
+  header:       "Available Azure Container Apps:"
```

Comment:
It seems the indentation is not fully aligned in this file. Suggest to re-indent some lines. For example, the string in this two lines need two more spaces indentation.